src/mock.tsx

```diff
@@ -0,0 +1,14 @@
+const fetchData = () => {
```

Comment:
what is the purpose of this file?

Reply:
Sorry, for confusion, this one needs to be deleted! I created it when starting unit tests and wanted to play with creating mock functions. Already deleted it)